File: 14-longest-common-prefix/longest-common-prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        sort(strs.begin(),strs.end());
        int a = strs.size();
        string x = strs[0], y = strs[a-1], ans="";
        for(int i=0 ; i<y.size() ; i++)
           {
              if(x[i]==y[i])
                   ans+=y[i];
              else
                 break;
           }

        return ans;
    }
};

File: 14-longest-common-prefix/longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        strs.sort()
        i=0
        ans=""
        while i<len(strs[0]):
            if strs[0][i]==strs[-1][i]:
                ans+=strs[0][i]
                i+=1
            else:
                 break
        return ans

        